### ISSUE TITLE
In the GIU show alias for site

### ISSUE THREAD

**Issue by act2281** [opendevshop/devshop#154]
There a lot of aliases in the list, and a few variations on the same site in some cases. Perhaps you could show in the GUI one of the drush aliases to make it easier to work with that site.

**Comment by jonpugh** [comment]
I've always thought that but never got around to it...

Here you go!

I realized there's a lot of info users need, so I included drush aliases under the new "ACCESS" modal window.

Does that make sense?

![screenshot from 2016-11-01 16-24-44](https://cloud.githubusercontent.com/assets/106420/19906077/be8e867c-a04f-11e6-8365-46eaed11c787.png)

![screenshot from 2016-11-01 16-24-49](https://cloud.githubusercontent.com/assets/106420/19906100/cf6ca15e-a04f-11e6-8910-c905ac9b9d4b.png)
